src/components/panel/panel.js

```diff
@@ -1306,10 +1386,22 @@ MdPanelRef.prototype._updatePosition = function(init) {
       this.panelContainer.addClass(MD_PANEL_HIDDEN);
     }
 
-    this.panelEl.css(MdPanelPosition.absPosition.TOP, positionConfig.getTop());
-    this.panelEl.css(MdPanelPosition.absPosition.BOTTOM, positionConfig.getBottom());
-    this.panelEl.css(MdPanelPosition.absPosition.LEFT, positionConfig.getLeft());
-    this.panelEl.css(MdPanelPosition.absPosition.RIGHT, positionConfig.getRight());
+    this.panelEl.css(
```

Comment:
Why formatting it? i think it was better one-lined, maybe using tabs for clarity
```
this.panelEl.css(MdPanelPosition.absPosition.TOP,    positionConfig.getTop());
this.panelEl.css(MdPanelPosition.absPosition.BOTTOM, positionConfig.getBottom());
this.panelEl.css(MdPanelPosition.absPosition.LEFT,   positionConfig.getLeft());
this.panelEl.css(MdPanelPosition.absPosition.RIGHT,  positionConfig.getRight());
```